Solutions/Windows Server DNS/Data Connectors/template_ASimDnsActivityLogs.JSON

```diff
@@ -0,0 +1,80 @@
+{
+    "id": "ASimDnsActivityLogs",
+    "title": "Windows DNS Events via AMA",
+    "publisher": "Microsoft",
+    "descriptionMarkdown": "The Windows DNS log connector allows you to easily filter and stream all analytics logs from your Windows DNS servers to your Microsoft Sentinel workspace using the Azure Monitoring agent (AMA). Having this data in Microsoft Sentinel helps you identify issues and security threats such as:\n- Trying to resolve malicious domain names.\n- Stale resource records.\n- Frequently queried domain names and talkative DNS clients.\n- Attacks performed on DNS server.\n\nYou can get the following insights into your Windows DNS servers from Microsoft Sentinel:\n- All logs centralized in a single place.\n- Request load on DNS servers.\n- Dynamic DNS registration failures.\n\nWindows DNS events are supported by Advanced SIEM Information Model (ASIM) and stream data into the ASimDnsActivityLogs table. [Learn more](https://docs.microsoft.com/azure/sentinel/normalization)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "ASimDnsActivityLogs",
+            "baseQuery": "ASimDnsActivityLogs\n| where EventProduct == \"DNS Server\""
```

Comment:
The EventProduct is generic and might be used by other sources using this table. Please add also a condition on EventVendor to the where clauses used.

Reply:
Hey @oshezaf, the query is updated as per this pr (https://dev.azure.com/msazure/One/_git/ASI-Portal/pullrequest/7562852?_a=files&path=/src/ASIExtension/Client/Data/DataConnectorsV3/Connectors/AmaConnectors/ASimDnsActivityLogsDataModel.ts), thanks.